### PR TITLE
DOC-510: Added Testnet decommission page

### PR DESCRIPTION
**Pull request by taylorharvey-r3**
Added Tesnet decommission page, moved some Testnet files, removed some empty Testnet-related files. This PR is built here (linked to OS 4.7 version of placeholder page): [https://tolkien.corda.net/docs/corda-os/4.7/deploy-to-testnet-index.html](https://tolkien.corda.net/docs/corda-os/4.7/deploy-to-testnet-index.html )

### CODE REVIEW

**Comment by taylorharvey-r3** [comment]
Adding a do not merge label pending publication date confirmation by Tiffany.

File: content/en/docs/corda-enterprise/4.7/network/corda-testnet-decommission.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+---
+
+# Corda Testnet
+
+{{< warning >}}
+To provide the best experience for users setting up a network, we have decided to decommission the use of Testnet effective April 16, 2021. Here we highlight other options that are available to users following the removal of Testnet.
+
+Based on your unique networking needs, we are offering a few different ways to get your network up and running.
+
+* [**Network Bootstrapper**](network-bootstrapper.md): An easy and flexible free tool for developers who are new to Corda. With Network Bootstrapper, you can set up remote nodes on any cloud platform.
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.7/network/corda-testnet-decommission.md:23]:
```suggestion
* [**Network Bootstrapper**](../network-bootstrapper.md): An easy and flexible free tool for developers who are new to Corda. With Network Bootstrapper, you can set up remote nodes on any cloud platform.
```

File: content/en/docs/corda-enterprise/4.6/network/corda-testnet-decommission.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+---
+
+# Corda Testnet
+
+{{< warning >}}
+To provide the best experience for users setting up a network, we have decided to decommission the use of Testnet effective April 16, 2021. Here we highlight other options that are available to users following the removal of Testnet.
+
+Based on your unique networking needs, we are offering a few different ways to get your network up and running.
+
+* [**Network Bootstrapper**](network-bootstrapper.md): An easy and flexible free tool for developers who are new to Corda. With Network Bootstrapper, you can set up remote nodes on any cloud platform.
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.6/network/corda-testnet-decommission.md:23]:
```suggestion
* [**Network Bootstrapper**](../network-bootstrapper.md): An easy and flexible free tool for developers who are new to Corda. With Network Bootstrapper, you can set up remote nodes on any cloud platform.
```

File: content/en/docs/corda-enterprise/4.5/network/corda-testnet-decommission.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+---
+
+# Corda Testnet
+
+{{< warning >}}
+To provide the best experience for users setting up a network, we have decided to decommission the use of Testnet effective April 16, 2021. Here we highlight other options that are available to users following the removal of Testnet.
+
+Based on your unique networking needs, we are offering a few different ways to get your network up and running.
+
+* [**Network Bootstrapper**](network-bootstrapper.md): An easy and flexible free tool for developers who are new to Corda. With Network Bootstrapper, you can set up remote nodes on any cloud platform.
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.5/network/corda-testnet-decommission.md:23]:
```suggestion
* [**Network Bootstrapper**](../network-bootstrapper.md): An easy and flexible free tool for developers who are new to Corda. With Network Bootstrapper, you can set up remote nodes on any cloud platform.
```

File: content/en/docs/corda-enterprise/4.4/network/corda-testnet-decommission.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+---
+
+# Corda Testnet
+
+{{< warning >}}
+To provide the best experience for users setting up a network, we have decided to decommission the use of Testnet effective April 16, 2021. Here we highlight other options that are available to users following the removal of Testnet.
+
+Based on your unique networking needs, we are offering a few different ways to get your network up and running.
+
+* [**Network Bootstrapper**](network-bootstrapper.md): An easy and flexible free tool for developers who are new to Corda. With Network Bootstrapper, you can set up remote nodes on any cloud platform.
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.4/network/corda-testnet-decommission.md:23]:
```suggestion
* [**Network Bootstrapper**](../network-bootstrapper.md): An easy and flexible free tool for developers who are new to Corda. With Network Bootstrapper, you can set up remote nodes on any cloud platform.
```

File: content/en/docs/corda-enterprise/4.3/corda-testnet-decommission.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+---
+
+# Corda Testnet
+
+{{< warning >}}
+To provide the best experience for users setting up a network, we have decided to decommission the use of Testnet effective April 16, 2021. Here we highlight other options that are available to users following the removal of Testnet.
+
+Based on your unique networking needs, we are offering a few different ways to get your network up and running.
+
+* [**Network Bootstrapper**](network-bootstrapper.md): An easy and flexible free tool for developers who are new to Corda. With Network Bootstrapper, you can set up remote nodes on any cloud platform.
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.3/corda-testnet-decommission.md:23]:
```suggestion
* [**Network Bootstrapper**](../network-bootstrapper.md): An easy and flexible free tool for developers who are new to Corda. With Network Bootstrapper, you can set up remote nodes on any cloud platform.
```

File: content/en/docs/corda-enterprise/4.2/corda-testnet-decommission.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+---
+
+# Corda Testnet
+
+{{< warning >}}
+To provide the best experience for users setting up a network, we have decided to decommission the use of Testnet effective April 16, 2021. Here we highlight other options that are available to users following the removal of Testnet.
+
+Based on your unique networking needs, we are offering a few different ways to get your network up and running.
+
+* [**Network Bootstrapper**](network-bootstrapper.md): An easy and flexible free tool for developers who are new to Corda. With Network Bootstrapper, you can set up remote nodes on any cloud platform.
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.2/corda-testnet-decommission.md:23]:
```suggestion
* [**Network Bootstrapper**](../network-bootstrapper.md): An easy and flexible free tool for developers who are new to Corda. With Network Bootstrapper, you can set up remote nodes on any cloud platform.
```

File: content/en/docs/corda-enterprise/4.1/corda-testnet-decommission.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+---
+
+# Corda Testnet
+
+{{< warning >}}
+To provide the best experience for users setting up a network, we have decided to decommission the use of Testnet effective April 16, 2021. Here we highlight other options that are available to users following the removal of Testnet.
+
+Based on your unique networking needs, we are offering a few different ways to get your network up and running.
+
+* [**Network Bootstrapper**](network-bootstrapper.md): An easy and flexible free tool for developers who are new to Corda. With Network Bootstrapper, you can set up remote nodes on any cloud platform.
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.1/corda-testnet-decommission.md:23]:
```suggestion
* [**Network Bootstrapper**](../network-bootstrapper.md): An easy and flexible free tool for developers who are new to Corda. With Network Bootstrapper, you can set up remote nodes on any cloud platform.
```

File: content/en/docs/corda-enterprise/4.0/corda-testnet-decommission.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+---
+
+# Corda Testnet
+
+{{< warning >}}
+To provide the best experience for users setting up a network, we have decided to decommission the use of Testnet effective April 16, 2021. Here we highlight other options that are available to users following the removal of Testnet.
+
+Based on your unique networking needs, we are offering a few different ways to get your network up and running.
+
+* [**Network Bootstrapper**](network-bootstrapper.md): An easy and flexible free tool for developers who are new to Corda. With Network Bootstrapper, you can set up remote nodes on any cloud platform.
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.0/corda-testnet-decommission.md:23]:
```suggestion
* [**Network Bootstrapper**](../network-bootstrapper.md): An easy and flexible free tool for developers who are new to Corda. With Network Bootstrapper, you can set up remote nodes on any cloud platform.
```

**Review by ivanterziev-r3 — CHANGES_REQUESTED** [review]
All looking good, just added fixes for the CE links to bootstrapper page

**Review by ivanterziev-r3 — APPROVED** [review]
LGTM

**Pull request merged**